addons/l10n_ar/models/account_move.py

```diff
@@ -2,6 +2,7 @@
 from odoo import models, fields, api, _
 from odoo.osv import expression
 from odoo.exceptions import UserError, RedirectWarning, ValidationError
+from odoo.tools.misc import formatLang
```

Comment:
You are importing formatLang but not using it.
I changed the language of the report to Spanish and found out that we are not properly managing the format of the amount (taking into account removing the t-options widget monetary)

Should use a comma instead of dot
![image](https://github.com/user-attachments/assets/50dfd06f-33cc-43db-82ef-b3dc1209de85)



 
We used the 16.0 PR to print the amounts in the new tax detail table properly.
This is necessary
![image](https://github.com/user-attachments/assets/b90d6428-c386-4a0a-9dc8-c552bfadf87f)